ENG-1946 - feat: introduce React package documentation (#143)
Basis-Theory/docs: source/includes/elements/react/_card_element.md

# CardElement

```jsx
import {
  CardElement
} from '@basis-theory/basis-theory-react/elements';

const MyForm = () => {  
  return <CardElement id="myCard" />;
}
```

> `CardElement` <a href="https://reactjs.org/docs/components-and-props.html" target="_blank">component</a> features a full credit card collector, by wrapping the [card element type](#element-types-card-element) functionality and taking care of its lifecycle.

Reviewer: ```suggestion
`CardElement` <a href="https://reactjs.org/docs/components-and-props.html" target="_blank">component</a> features a full credit card form, wrapping the [card element type](#element-types-card-element) functionality and taking care of the React lifecycle.
```
